File: langhuan-service/src/main/java/com/langhuan/model/pojo/RagMetaData.java
package com.langhuan.model.pojo;

import lombok.Data;

@Data
public class RagMetaData {
    private String filename;
    private String fileId;
    private String groupId; // HACK: 改动字段名注意搜索：filterExpression
    private int rank; // 0-100的整数。 HACK: 改动字段名注意搜索：filterExpression
}

File: langhuan-server/src/main/kotlin/com/langhuan/model/pojo/RagMetaData.kt
package com.langhuan.model.pojo

import lombok.Data

@Data
data class RagMetaData(
    var filename: String? = null,
    var fileId: String? = null,
    var groupId: String? = null, // HACK: 改动字段名注意搜索：filterExpression
    var rank: Int = 0 // 0-100的整数。 HACK: 改动字段名注意搜索：filterExpression
)
